### PR TITLE
add external components to enterprise implementation docs

### PR DESCRIPTION
**Pull request by Hesperide**
* external db
* ingress
* external logging

### CODE REVIEW

File: docs/enterprise-setup/implementation-guide.md [hunk 1]
```diff
@@ -81,11 +81,162 @@ To configure basic auth (deploy without SSO), remove the entire `auth:` section
...
+
+```yaml
+externalDatabase:   
+    host: ## Database host
+    user: ## Non-root username for the Airbyte database
+    database: db-airbyte ## Database name
+    port: 5432 ## Database port number 
+```
+
+For the non-root user's password which has database access, you may use `password`, `existingSecret` or `jdbcUrl`. We recommend using `existingSecret`, or injecting sensitive fields from your own external secret store. Each of these parameters is mutually exclusive:
```
Comment (colesnodgrass) [inline docs/enterprise-setup/implementation-guide.md:107]: > injecting sensitive fields

Are we going to document (could be in the future) how we recommend doing this?

File: docs/enterprise-setup/implementation-guide.md [hunk 2]
```diff
@@ -81,11 +81,162 @@ To configure basic auth (deploy without SSO), remove the entire `auth:` section
...
+  database:
+    secretName: "airbyte-enterprise-airbyte-secrets" ## The name of the existing Kubernetes secret entered above.
+    secretValue: "DATABASE_PASSWORD" ## The Kubernetes secret key entered above.
+    host: "localhost" ## Database host entered above.
+    port: "5432" ## Database port entered above.
+```
+
+If you've used `password` or `jdbcUrl` above instead using a pre-existing Kubernetes secret, then your `secretName` will be `{release-name}-airbyte-secrets`, and `secretValue` will be `DATABASE_PASSWORD`.
+
+#### Configuring External Logging
```
Comment (colesnodgrass) [inline docs/enterprise-setup/implementation-guide.md:134]: Don't we also need external state storage configured as well.

File: docs/enterprise-setup/implementation-guide.md [hunk 3]
```diff
@@ -81,11 +81,162 @@ To configure basic auth (deploy without SSO), remove the entire `auth:` section
...
+    ...
+    password: ## Password for non-root database user
+    existingSecret: ## The name of an existing Kubernetes secret containing the password.
+    existingSecretPasswordKey: ## The Kubernetes secret key containing the password.
+    jdbcUrl: "jdbc:postgresql://<user>:<password>@localhost:5432/db-airbyte" ## Full database JDBC URL. You can also add additional arguments.
+```
+
+The optional `jdbcUrl` field should be entered in the following format: `jdbc:postgresql://localhost:5432/db-airbyte`. We recommend against using this unless you need to add additional extra arguments can be passed to the JDBC driver at this time (e.g. to handle SSL).
+
+3. Finally, add this configuration into the `global` section of the `charts/airbyte/values.yaml` as well:
```
[the remark this answers is not in the document]
Comment (perangel) [reply docs/enterprise-setup/implementation-guide.md:120]: I was looking at this earlier as well to try and explain it to Alex, it reads like it just evolved over time and we never fully reconciled the configuration. But it is very confusing, either way.

File: docs/enterprise-setup/implementation-guide.md (hunk 2 above)
Comment (perangel) [reply docs/enterprise-setup/implementation-guide.md:134]: This is where things seem to fall apart for me. Right now it sort of reads like the state configuration is coupled to the configuration of logging, as it is the same credentials that you would presumably be using for both, but you can certainly use one product for state and another for logging.

That said, we prob to need an explicit section for configuration of state storage

**Comment by perangel** [comment]
This is great Alex. Overall it looks good, I would maybe make a small NIT re: phrasing. 

Thoughts on "we recommend using X" vs "we advise against Y".
e.g. We recommend configuring external logging storage using .... The default configuration is intended for non-production use.

It may just be me, but the "advise against" feels like there is some danger or that it is inherently bad.

**Review by perangel — APPROVED** [review]

File: docs/enterprise-setup/implementation-guide.md (hunk 1 above)
Comment (Hesperide) [reply docs/enterprise-setup/implementation-guide.md:107]: We have to :)

File: docs/enterprise-setup/implementation-guide.md (hunk 2 above)
Comment (Hesperide) [reply docs/enterprise-setup/implementation-guide.md:134]: I'm happy to add a section for state storage once it's clear what needs to be configured :)

**Pull request merged**